frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.component.tsx

```diff
@@ -15,7 +15,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
In a ticket with a pin and a boolean property:
- create the pin; (the ticket update toast will show up)
- switch the boolean to true; (don't click anything else, so the toast does not show up)
- delete the pin;

This will trigger an error

https://github.com/3drepo/3drepo.io/assets/43547703/3279d32c-1efc-4209-9e1c-87f4ed3cc33f

Reply:
I have made an [issue](https://github.com/3drepo/3drepo.io/issues/4621) for this as it is already happening on staging, and happens with more than just pins